symfony/src/Entity/MeasurementHistory.php

```diff
@@ -24,9 +24,9 @@ class MeasurementHistory
     private $evaluation;
 
     /**
-     * @ORM\Column(type="integer")
+     * @ORM\OneToOne(targetEntity="App\Entity\User", cascade={"persist", "remove"})
```

Comment:
Shouldn't this be a ManyToOne relationship since a user can create many measurement histories?